src/test/javascript/spec/component/course/course-update.component.spec.ts

```diff
@@ -487,4 +492,18 @@ describe('Course Management Update Component', () => {
             return fixture.debugElement.nativeElement.querySelector('#delete-course-icon');
         }
     });
+
+    describe('changeOrganizations', () => {
+        it('should allow adding / removing organizations if admin', () => {
+            jest.spyOn(accountService, 'isAdmin').mockReturnValue(true);
+            fixture.detectChanges();
+            expect(comp.isAdmin).toBeTrue();
+        });
+
+        it('should not allow adding / removing organizations if not admin', () => {
+            jest.spyOn(accountService, 'isAdmin').mockReturnValue(false);
+            fixture.detectChanges();
+            expect(comp.isAdmin).toBeFalse();
+        });
```

Comment:
You can also make sure that the correlating button is shown/hidden. Otherwise you are more or less only testing the variable assignment